graph-manager-docs/source/setup-analytics.md

```diff
@@ -33,47 +32,38 @@ server.listen().then(({ url }) => {
 });
 ```
 
-For advanced configuration options to set up logging, error filtering, and client-aware metrics reporting take a look at our [server documentation](https://www.apollographql.com/docs/apollo-server/features/metrics.html).
-
-## Other servers
+Alternatively, you can assign your API key to the `ENGINE_API_KEY` environment variable in the environment where Apollo Server will run.
 
-There are 2 ways to send metrics data from your server to Engine:
+This is the only change required to begin sending traces to Graph Manager. For advanced configuration options, see [Metrics and logging](https://www.apollographql.com/docs/apollo-server/features/metrics/).
```

Comment:
capital L in logging? 🤷‍♂

Reply:
Sure looks like it should be, but we don't title case articles currently